src/content/community/meetups.md

```diff
@@ -136,6 +136,9 @@ Do you have a local React.js meetup? Add it here! (Please keep the list alphabet
 ## Spain {/*spain*/}
 * [Barcelona](https://www.meetup.com/ReactJS-Barcelona/)
 
+## Sri Lanka {/*sri-lanka*/}
+* [Colombo](https://www.javascriptcolombo.com/)
```

Comment:
The links on the website doesnt seem to work?

Reply:
Thanks for taking the time to review the PR @harish-sethuraman.

We are still in the process of building the website.
I've pushed an update to the site that would direct the registration to our meetup group (https://www.meetup.com/JavaScript-Colombo/)

Please check now and approve.

Reply:
Thanks for taking the time to review the PR @harish-sethuraman.

We are still in the process of building the website.
I've pushed an update to the site that would direct the registration to our meetup group (https://www.meetup.com/JavaScript-Colombo/)

Please check now and approve.